fnmatch.go

```
@@ -0,0 +1,169 @@
+package editorconfig
+
+import (
+	"fmt"
+	"log"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+func FnmatchCase(pattern, name string) bool {
+	p, _ := translate(pattern)
```

Are you sure you want to ignore the error here?

---

This function should probably return `(bool, error)` instead, to allow returning this error.

---

Good point.